ydb/core/driver_lib/run/ya.make

```diff
@@ -1,5 +1,15 @@
 LIBRARY(run)
 
+IF (OS_WINDOWS)
+    CFLAGS(
+        -DKIKIMR_DISABLE_S3_OPS
```

Comment:
I thought that `KIKIMR_DISABLE_S3_OPS` flag would be inherited from [`ydb/apps/ydbd/ya.make`](https://github.com/ydb-platform/ydb/blob/4d7315945d3c4786aaf3677b406cedb0c5c39ca6/ydb/apps/ydbd/ya.make#L18). However, this was not the case in my tests, so I included it here.